sidecar/sidecar.go

```diff
@@ -68,15 +66,9 @@ func Main() {
 }
 
 func sidecarCommand(context *cli.Context) error {
-	if os.Getenv("A8_CHILD") == "" {
-		fmt.Println(os.Args[1:])
-		cmd := exec.Command("a8sidecar", os.Args[1:]...)
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-		cmd.Env = append([]string{"A8_CHILD=yes"}, os.Environ()...)
-		// listen for SIGCHLDs from any of the supervised processes and any orphans spun off from those
-		go supervisor.ReapZombies()
-		return cmd.Run()
+	// when PID=1, launch new sidecar process and assume init responsibilities
+	if os.Getpid() == 1 {
```

Comment:
coding convention that golint checks against, it complains when there are return statements in an if/else
```
if PID==1 {

    //do something
    return value
} else {
    //do something else
    return value
}
```
specifically: `amalgam8/sidecar/sidecar.go:75:9: if block ends with a return statement, so drop this else and outdent its block`